examples/with_ha-sap-terraform-deployments/pillar.example.cluster_cloud

```diff
@@ -0,0 +1,64 @@
+cluster:
```

Comment:
I wouldn't personally add this file.
It has a lot of references to the `ha-sap-terraform-deployments` project. Actually, it would only work in that project. I would just keep this in that project